pkg/apis/garden/v1beta1/helper/helpers.go

```diff
@@ -80,9 +80,9 @@ func GetShootCloudProvider(shoot *gardenv1beta1.Shoot) (gardenv1beta1.CloudProvi
 	return DetermineCloudProviderInShoot(shoot.Spec.Cloud)
 }
 
-// IsShootHibernated checks if the given shoot is hibernated.
+// IsShootHibernated checks if the given shoot's desired state is hibernated.
 func IsShootHibernated(shoot *gardenv1beta1.Shoot) bool {
```

Comment:
Can you also rename this func like `HibernationDesired` or maybe you have something better in mind :)